packages/framework/src/Model/AdminNavigation/SideMenuBuilder.php

```diff
@@ -230,7 +230,8 @@ protected function createSettingsMenu(): ItemInterface
 
         $communicationMenu = $menu->addChild('communication', ['label' => t('Communication with customer')]);
         $communicationMenu->addChild('mail_settings', ['route' => 'admin_mail_setting', 'label' => t('Email settings')]);
-        $communicationMenu->addChild('mail_templates', ['route' => 'admin_mail_template', 'label' => t('Email templates')]);
+        $mailTemplates = $communicationMenu->addChild('mail_templates', ['route' => 'admin_mail_template', 'label' => t('Email templates')]);
+        $mailTemplates->addChild('edit_template', ['route' => 'admin_mail_edit', 'label' => t('Editing email template'), 'display' => false]);
```

Comment:
why does it have to be here? Due to breadcrumb navigation?

Reply:
Yes. 
Without this entry, on the edit page of the mail template (new one) breadcrumb show only root level. With this, it's a full breadcrumb path.